.travis.yml

```diff
@@ -49,6 +49,21 @@ jobs:
       script: travis_wait docker-compose run -e FORM_FACTOR=huge test-ci
     - name: enormous
       script: travis_wait docker-compose run -e FORM_FACTOR=enormous test-ci
+    - stage: compile - lowlight theme
+      script: travis_wait docker-compose run test-ci-lowlight npm run compile:lowlight
+    - stage: wdio - lowlight theme
+      name: tiny
+      script: travis_wait docker-compose run -e THEME=clinical-lowlight-theme -e FORM_FACTOR=tiny test-ci-lowlight
+    - name: small
+      script: travis_wait docker-compose run -e THEME=clinical-lowlight-theme -e FORM_FACTOR=small test-ci-lowlight
+    - name: medium
+      script: travis_wait docker-compose run -e THEME=clinical-lowlight-theme -e FORM_FACTOR=medium test-ci-lowlight
+    - name: large
+      script: travis_wait docker-compose run -e THEME=clinical-lowlight-theme -e FORM_FACTOR=large test-ci-lowlight
+    - name: huge
+      script: travis_wait docker-compose run -e THEME=clinical-lowlight-theme -e FORM_FACTOR=huge test-ci-lowlight
```

Comment:
we are using the lowlight test script, we can drop the THEME env.

Reply:
Yeah my bad, addressed it here : https://github.com/cerner/terra-core/pull/2758/commits/f0b4184ca3d941e38b02ceb6ee8fb5f386d9efd2